lib/constants.ts

```diff
@@ -19,19 +19,20 @@ export enum TimetableEventKey {
   PlanNotAdopted = "plan-not-adopted",
 }
 
-export type StatusChangeEventsKey = Extract<
-  TimetableEventKey,
-  | TimetableEventKey.PlanFoundUnsound
-  | TimetableEventKey.PlanWithdrawn
-  | TimetableEventKey.PlanPaused
-  | TimetableEventKey.PlanNotAdopted
->;
+export const statusChangeEvents = [
+  TimetableEventKey.PlanFoundUnsound,
+  TimetableEventKey.PlanWithdrawn,
+  TimetableEventKey.PlanPaused,
+  TimetableEventKey.PlanNotAdopted,
+] as const;
```

Comment:
Could we use this to spread into the set we use for filtering the default events below?

Reply:
done :)